pkgs/data/themes/flat-remix-gtk/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flat-remix-gtk";
+  version = "20200718";
+
+  src = fetchFromGitHub {
+    owner = "daniruiz";
+    repo = pname;
+    rev = version;
+    sha256 = "04hv8b512ldjrn4gigna24gdf3finm7j98995npm7lyhnfc4smgf";
+  };
+
+  dontBuild = true;
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    inherit (src.meta) homepage;
```

Comment:
It is more helpful to explicitly write the homepage here. It would allow copying and pasting in a browser easily, for instance.

_Edited_: There is also a more specific page for it (https://drasite.com/flat-remix-gtk) that would be appropriate as a homepage. I suggest to use it.